wordpress-mobile/WordPress-iOS
# Show Reader content in a WebView

Fixes #14183

This adds the additional architecture for rendering post content inside a WebView. Some important things:

* It's all under a FeatureFlag that is *always* `false`
* My main goal is to keep the ViewController as simple as possible
* The coordinator should handle service and other stuff

### To test

First, you need to override the feature flag value. Open any site > tap on your profile (top-right icon) > tap "App Settings" > tap "Debug" (last option) > Toggle "Reader content displayed in a WebView" ON. [Like this](https://user-images.githubusercontent.com/7040243/82946518-e0842200-9f74-11ea-82b3-8108ba877568.png).

Then, open the Reader and tap some articles, you should see the content being displayed there (with no style at all).

PR submission checklist:
